scripts/create_likelihood_database.py

```diff
@@ -26,6 +26,12 @@ def parse_args():
     parser.add_argument("--output", "-o",
                         default="limit",
                         help="Name of the output plot without file extension")
+    parser.add_argument("--x_var", "-x",
```

Comment:
I would prefer to have this argument with a dash instead of an underscore:
```suggestion
    parser.add_argument("--x-var", "-x",
```